workspace/index.js

```diff
@@ -119,11 +119,6 @@ module.run(function ($log, $q, $http, ImpacLinking, ImpacAssets, ImpacRoutes, Im
     }
   });
 
-  // Link Impac! Assets
```

Comment:
Because the library is providing it by default - the whole point of these changes